data_and_models/pipelines/ner/transformers_vs_spacy/spacy/eval.py

```diff
@@ -16,7 +14,7 @@
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
-
+"""Evaluation script for NER models."""
```

Comment:
Ditto — I would keep the docstring above.

Reply:
see above